testpilot/settings.py

```diff
@@ -509,7 +509,8 @@ def lazy_langs():
     "https://code.cdn.mozilla.net/fonts/fira.css":
         "sha384-APhs/OUouhH+ZbBANL3+7a5J1sVSYyfUhGxTWiDMkTuEIO5fXWZonzNEj+CagDB7",
 }
-STATICINTEGRITY_CACHE_TTL = 30
+STATICINTEGRITY_CACHE_TTL = config('STATICINTEGRITY_CACHE_TTL',
```

Comment:
could you explain this change somewhere? is this just removing the hardcoded value?